curriculum/challenges/english/25-front-end-development/workshop-magazine/614385513d91ae5c251c2052.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 614385513d91ae5c251c2052
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+In this workshop, you'll create a magazine page using a press release for an old version of fCC's curriculum.
```

Comment:
If anyone has suggestions to improve the wording it would be great.

Reply:
In this workshop, you will create a magazine page using a press release from a previous version of freeCodeCamp's curriculum.